instrumentation/spring/spring-webmvc-3.1/spring-webmvc-3.1-auto/spring-webmvc-3.1-auto.gradle

```diff
@@ -39,9 +39,9 @@ dependencies {
 
   testImplementation group: 'org.spockframework', name: 'spock-spring', version: "$versions.spock"
 
-  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: '1.5.17.RELEASE'
-  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: '1.5.17.RELEASE'
-  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: '1.5.17.RELEASE'
+  testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: '2.3.1.RELEASE'
```

Comment:
I am not sure about this change. We usually test the oldest supported version by default and let `latestDepTests` verify the latest supported version.

Reply:
Is it changed to 2.x stream by latestDep? I had to make a few changes around test code (AppConfig, ServletFIlteTest) to make it work (spring boot 2 is in some aspects not compatible with 1.x stream). I assumed it's worth upgrading (in baseline) as boot 2 has been around for more than 2 years now. Let me know what you think about that.

Reply:
I just realised that SB 2.3.1 most certainly does not use Spring MVC 3.1. So after these changes we don't test if we actually support 3.1. And as there are API changes affecting tests, it means we should test them separately but certainly should test them both. So I propose to leave this with old version as it was. And we should add a separate test with more recent versions of Spring MVC. You don't have to add that new test right away, you can add a follow up issue.

Reply:
Per slack conversation - made slight adjustments for the code to get both boot 1.x and 2.x compatibility.